README.md

```diff
@@ -101,6 +101,20 @@ tuist generate # Generates Xcode project
 3.  Open `tuist.xcodeproj`.
 4.  Have fun 🤖
 
+## Testing
```

Comment:
Good idea to document this 👍. What do you think about moving this section into a testing page inside [this folder](https://github.com/tuist/documentation/tree/master/contribution). We are aiming for keeping all the documentation organised in that repository.

Reply:
@kwridan 's idea & execution 👍